src/coercers.test.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  DECIMAL_NUMBERS,
+  HEX_STRINGS,
+  NEGATIVE_INTEGERS,
+  POSITIVE_INTEGERS,
+} from './__fixtures__';
+import { createBigInt, createBytes, createHex, createNumber } from './coercers';
+import { add0x } from './hex';
+import { bytesToHex, hexToBytes } from './bytes';
+
+describe('createNumber', () => {
```

Comment:
Should we test that this accepts 0x-prefixed hex strings? And also treats strings that look like numbers as decimals?